CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ to use the newest tag with new release
 ### Added
 
 - `cartridge-replicasets` tag to the membership stage
+- `cartridge_wait_buckets_discovery` parameter to wait for instance to discover buckets
+- `instance_discover_buckets_timeout` parameter to configure time in seconds
+  to wait for instance to discover buckets
```

Comment:
Why there are two similar parameters?

Reply:
It's not similar options. First enable or disable waiting for buckets discovering. Second configure timeout of buckets discovering, if it's enabled